assembly/pom.xml

```diff
@@ -66,9 +66,21 @@
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
-      <artifactId>spark-sql_${scala.binary.version}</artifactId>
+      <artifactId>xsql-sql_${scala.binary.version}</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-sql_${scala.binary.version}</artifactId>
+      <version>${spark.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-catalyst_${scala.binary.version}</artifactId>
+      <version>${spark.version}</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
Why there need to add catalyst dependency?

Reply:
If we don't mark spark-sql as provided, final assembly jars will contains spark-sql as well as xsql-sql module at same time which will lead to Class conflict.

Reply:
> Why there need to add catalyst dependency?

The problem of spark-catalyst is the same as spark-sql. If we don't mark spark-catalyst as provided, final assembly jars will contains spark-catalyst as well as xsql-catalyst module at same time which will lead to Class conflict.